hudi-client/hudi-java-client/src/test/java/org/apache/hudi/table/action/commit/TestJavaCopyOnWriteActionExecutor.java

```diff
@@ -413,6 +415,7 @@ public void testInsertUpsertWithHoodieAvroPayload() throws Exception {
         .withEngineType(EngineType.JAVA)
         .withPath(basePath)
         .withSchema(schema.toString())
+        .withMarkersType(MarkerType.DIRECT.name())
```

Comment:
Any plan/ticket to convert all these tests to timeline based markers? Or by design these still have to be direct type?